### PR TITLE
Don't match multiple time the same string

### PR DESCRIPTION
**Pull request by Taiki-San**
### What does this PR do?
The phrase_match operator match substrings. Therefore, matching variants of the same string is unecessary and a waste of resources.

### Checklist
- [ ] Did you added the `test_vectors` for your rules
- [ ] Did you tested it on your local machine first
- [ ] Did you tested it on staging

### CODE REVIEW

File: rules/recommended/crs-930-120.yaml [hunk 1]
```diff
@@ -1041,16 +1041,11 @@ conditions:
         - etc/timezone
         - etc/modules
         - etc/passwd
-        - etc/passwd~
```
Comment (netcode) [inline rules/recommended/crs-930-120.yaml:1044]: I think its better to remove it from the RCE and keep it here in LFI, no?

Comment (netcode) [reply rules/recommended/crs-930-120.yaml:1044]: OK, I saw the other PR.

File: rules/recommended/crs-930-120.yaml [hunk 2]
```diff
@@ -1041,16 +1041,11 @@ conditions:
...
-        - etc/passwd-
         - etc/shadow
-        - etc/shadow~
-        - etc/shadow-
```
Comment (netcode) [inline rules/recommended/crs-930-120.yaml:1048]: Does `phrase_match` do strict match? or they do partial match?

Comment (Taiki-San) [reply rules/recommended/crs-930-120.yaml:1048]: Partial matches, which causes other problems ^^'

**Review by netcode — APPROVED** [review]

**Pull request merged**